### ISSUE TITLE
Enum and wrong naming convention 

### ISSUE THREAD

**Issue by quolpr** [dotansimha/graphql-code-generator#1569]
**To Reproduce**

1. My GraphQL schema:

```graphql
enum FeedbackPollShowOption {
  ALWAYS
  HIDE_AFTER_SUBMIT
}
```

2. My `codegen.yml` config file:

```yml
documents: ./src/**/*.graphql
overwrite: true
generates:
  ./src/queries.tsx:
    plugins:
      - add: // tslint:disable
      - typescript
      - typescript-operations
      - typescript-react-apollo
    config:
      noNamespaces: true
      avoidOptionals: false
      scalars:
        UUID4: string
        DateTime: string
        Json: string
        Decimal: string
        Date: string
```

**Current behavior**
```
export enum FeedbackPollShowOption {
  Always = "ALWAYS",
  Hide_After_Submit = "HIDE_AFTER_SUBMIT"
}

```

**Expect Behaviour**
```
export enum FeedbackPollShowOption {
  Always = "ALWAYS",
  HideAfterSubmit = "HIDE_AFTER_SUBMIT"
}
```
As it was before.

**Environment:**

- `@graphql-codegen/...`: `1.0.4`

**Comment by quolpr** [comment]
`1.0.0` has the same behavior 🤔

**Comment by dotansimha** [comment]
Caused by the fact that we would like to keep the underscore `_`  when you defined it, and apply the naming conventing without effecting it. 
@kamilkisiela @ardatan what do you think?

**Comment by dotansimha** [comment]
Fixed in 1.0.5.
